Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -62,40 +66,71 @@ namespace AzTestRunner
             return INCORRECT_USAGE;
         }
 
+        // copy over the command line args to a vector so we can expand with any arguments from file
+        std::vector<std::string> arguments(argv, argv + argc);
+
         // capture positional arguments
         // [0] is the program name
-        std::string lib = argv[1];
-        std::string symbol = argv[2];
+        std::string lib = arguments[1];
+        std::string symbol = arguments[2];
 
-        // shift argv parameters down as we don't need lib or symbol anymore
-        AZ::Test::RemoveParameters(argc, argv, 1, 2);
+        // shift args parameters down as we don't need lib or symbol anymore
+        arguments.erase(std::next(arguments.begin(), 1), std::next(arguments.begin(), 3));
```

Comment:
question: Just wanted to sanity check you've thoroughly tested this and the behavior is the same as before

Reply:
Instead of using a costly erase function, instead a fixed_vector can be made out of the parts of the span containing the zero'th executable argument and the remaining arguments
```suggestion
        AZStd::fixed_vector<std::string_view> argStorage(arguments.begin(), arguments.begin() + 1);
        argStorage.insert(argStorage.end(), arguments.begin() + 3, arguments.end());
```